src/block/block_ckpt.c

```diff
@@ -402,14 +402,17 @@ __ckpt_verify(WT_SESSION_IMPL *session, WT_CKPT *ckptbase)
     return (0);
 }
 
+/* At the default granularity, this is enough for blocks in a 2G file. */
+#define WT_BLOCK_MODS_LIST_MIN 128 /* Initial bits for bitmap. */
```

Comment:
Looks like it should be renamed too? Can you grep "WT_BLOCK_MOD" to make sure we are not missing anything?

Reply:
Why? It's not in the checkpoint module.

Reply:
I understand that it's connected with the one that's got renamed. But it'd break the naming rules. Or we need to move it to the checkpoint module.

Reply:
Sorry - I misread. I did not see it was the **block**_ckpt.c file - my bad.

Reply:
No actually it could be a good idea to move this into the checkpoint module despite there's just one use of it.

@sueloverso what do you think?

Reply:
Its only use is in `__ckpt_mod_blkmod_entry` which is inside that file - I guess we could leave it. No need to expose it further if we only use it once.

Reply:
This is a matter of design intention - whether it’s part of the `checkpoint` interface or an internal detail of `block`.

If it’s part of the interface, its usage frequency doesn’t matter. Even if it’s used only once or not at all, it remains an interface component.